notebooks/tfx_pipelines/pipeline/solutions/tfx_pipeline_vertex.ipynb

```diff
@@ -0,0 +1,391 @@
+{
```

Comment:
<p>Nit: consider model.py explanation as implements the TensorFlow model code and training logic for the Trainer component (singular).</p><p>Nit: Consider "on Vertex" -&gt; Vertex Pipelines just to clarify which service is executing the pipeline run.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/139/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done!</p>